INSTALL_MACOS.md

```diff
@@ -58,13 +58,18 @@ It is also recommended to add the  following line to `/opt/local/etc/macports/va
 -x11 +no_x11 +bash_completion +no_gnome +quartz +natron
 ```
 
-#### Notes for OS X 10.6 Snow Leopard
+#### Notes for MacOS 10.6 Snow Leopard
 
-The libtool that comes with OS X 10.6 Snow Leopard does not work well with clang-generated binaries, so on this system you may have to substitute it with the libtool provided by MacPort's `cctools` package, using `sudo mv /usr/bin/libtool /usr/bin/libtool.orig; sudo mv /Developer/usr/bin/libtool /Developer/usr/bin/libtool.orig; sudo ln -s /opt/local/bin/libtool /usr/bin/libtool; sudo ln -s /opt/local/bin/libtool /Developer/usr/bin/libtool`
+The libtool that comes with MacOS 10.6 Snow Leopard does not work well with clang-generated binaries, so on this system you may have to substitute it with the libtool provided by MacPort's `cctools` package, using `sudo mv /usr/bin/libtool /usr/bin/libtool.orig; sudo mv /Developer/usr/bin/libtool /Developer/usr/bin/libtool.orig; sudo ln -s /opt/local/bin/libtool /usr/bin/libtool; sudo ln -s /opt/local/bin/libtool /Developer/usr/bin/libtool`
 
 #### Install Ports
 
-Install the required packages:
+Now, use libjpeg-turbo instead of jpeg:
```

Comment:
Not required anymore, MacPorts comes with libjpeg-turbo. Did you get that from an older version of README.md?

Reply:
Yes, it's fixed now.